adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/MqttProtocolAdapterProperties.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.adapter.mqtt;
+
+import org.eclipse.hono.config.ProtocolAdapterProperties;
+
+/**
+ * Properties for configuring an Mqtt based protocol adapter.
+ *
+ */
+public class MqttProtocolAdapterProperties extends ProtocolAdapterProperties {
+
+    private static final long DEFAULT_COMMAND_ACKNOWLEDGEMENT_TIME_OUT = 100L;
```

Comment:
this needs to be at least *protected* in order for the link to this field from the method JavaDoc to succeed ...

Reply:
Changed.